docs/dsl_resource.md

```diff
@@ -37,14 +37,25 @@ require a higher version.
 
 The following attributes can be configured:
 
-* name - Identifier of the resource (required)
-* desc - Description of the resource (optional)
-* example - Example usage of the resource (optional)
+<dl>
+  <dt>name</dt>
+  <dd>Identifier of the resource (required)</dd>
+  <dt>desc</dt>
+  <dd>Description of the resource (optional)</dd>
+  <dt>example</dt>
+  <dd>Example usage of the resource (optional)</dd>
+  <dt>supports</dt>
+  <dd><small>(InSpec 2.0+)</small> Platform restrictions of the resource (optional)</dd>
+</dl>
 
```

Comment:
Our house style avoids HTML in Markdown.

On supports: We use platform for OS/AWS/Azure. This usage appears to be for InSpec's semver. I suggest:
- supports - InSpec version restrictions for this resource.

Reply:
Actually, `supports` is supposed to indicate what platform a resource is known to work with. Where a lot of resources are written to `skip_resource` under certain platform-specific circumstances (e.g., `etc_fstab` isn't going to work on Windows), this new feature simplifies that duplication of checking the platform so it's an attribute like `desc` or `example`.

I see where it was confusing about it representing the inspec version the resource supports. I was just trying to differentiate that it was a feature only supported in InSpec 2.x, not the 1.x line that many people are still working off of (if using ChefDK, for instance).